robot/client/client.go

```diff
@@ -935,3 +936,15 @@ func (rc *RobotClient) StopAll(ctx context.Context, extra map[resource.Name]map[
 	_, err := rc.client.StopAll(ctx, &pb.StopAllRequest{Extra: e})
 	return err
 }
+
+// ModuleLog sends a log entry to the server. To be used by Golang modules wanting to log over gRPC and not
+// by normal Golang SDK clients.
```

Comment:
It's possible to not make this public API if we're really concerned with Golang SDK users (or module authors) improperly using it. It was easier for me to expose a new method here, though.